docs/UserGuide.md

```diff
@@ -90,69 +90,40 @@ May have irreversible effect when used. Backup and caution is recommended.
 </div>
 
 
-### 1.2 How to use FitEgo
+### 1.2 Overview
 
-FitEgo supports a lot of [commands](#5-command-summary), and it might seem daunting to learn all at once. This section aims to solve that, and guide you to understand how most FitEgo's commands are structured!
+You're a fitness instructor.
 
-There are 3 major entities in FitEgo (clients, sessions, and schedules), and 5 major verbs (add, edit, delete, view, list).
+Your clients' details, clients' progress, clients' payment status, and your own schedules are spread across 3 or 4 different applications. 
```

Comment:
"Your records of your clients' details, training progress, payment status and your own timetable are spread across 3 or 4 different applications."